test/test_core.py

```diff
@@ -49,39 +49,41 @@ def test_core_search(run_command, httpserver):
     result = run_command(f"core search retrokit --all --additional-urls={url}")
     assert result.ok
     lines = [l.strip().split() for l in result.stdout.strip().splitlines()]
-    assert 11 == len(lines)
     assert ["Updating", "index:", "package_index.json", "downloaded"] in lines
     assert ["Updating", "index:", "package_index.json.sig", "downloaded"] in lines
     assert ["Retrokits-RK002:arm", "1.0.5", "RK002"] in lines
     assert ["Retrokits-RK002:arm", "1.0.6", "RK002"] in lines
+    header_index = lines.index(["ID", "Version", "Name"])
+    assert 2 == len(lines[header_index + 1 :])
```

Comment:
```suggestion
    assert 2 == len(lines[(header_index + 1):])
```
What do you think about doing this instead, and then leaving the `E203` flake8 check intact? It seems easier to understand to me

Reply:
That wouldn't work, `black` would format it as `lines[(header_index + 1) :]` so `flake8` complains anyway.